### PR TITLE
Mnist Dataset classification

### PR DESCRIPTION
**Pull request by yagyesh-bobde**
## Issue Title : Mnist Dataset classification

- Info about the related issue (Aim of the project)

- [x] SWOC Participant 
- [x] Contributor

Closes: #6

### Describe the add-ons or changes you've made
I have created a machine learning project for image classification using MNIST dataset. I have compared 3 different models and accuracies.

## How Has This Been Tested?
I have created the jupyter notebook and trained each model myself.

## Checklist:

- [x ] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by yagyesh-bobde** [comment]
ok

**Comment by yagyesh-bobde** [comment]
actually i wanted to ask like do i keep the .vscode and .ipynb_checkpoints file or should i delete them

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Follow the Demo-Project Folder and remove the unnecessary files @yagyesh-bobde 

Here's the link for the Demo-Project Folder, https://github.com/abhisheks008/ML-Crate/tree/main/Project-Demo-Folder

**Comment by abhisheks008** [comment]
Then no need to put the Images folder @yagyesh-bobde

**Comment by yagyesh-bobde** [comment]
did the changes ..... PTAL

File: MNIST Dataset Classification/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+*This folder will be for storing the model created for the ML project. The file format should be in .ipynb.*
```
Comment (abhisheks008) [inline MNIST Dataset Classification/Model/README.md:1]: remove this line.

File: MNIST Dataset Classification/Model/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| ---| --- |
+| 1. Logistic Regression | 84.4% | 
+| 2. Random Forest | 95% |
+| 3. 3 layer NN | 97% | 
+
+I also tried using 4 layers and played around with the number of hidden units and in some cases the accuracy did increase a bit but at the same time the computation time also increased, and by a lot if I may add. 
+
+So in conclusion I ended up selecting the 3 layer NN with 64 hidden units as the most optimal option.
+
+![image](https://user-images.githubusercontent.com/90238742/145832775-c7c5cc8b-4886-4635-8107-fe105e96cd37.png)
```
Comment (abhisheks008) [inline MNIST Dataset Classification/Model/README.md:51]: This image is not visible. Create Images folder and upload the image there, then hyperlink the image from the Images folder to the `Model/README.md` file. 
There will be no README.md file in the Images folder

File: MNIST Dataset Classification/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+numpy
```
Comment (abhisheks008) [inline MNIST Dataset Classification/requirements.txt:1]: Add the versions of the libraries

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

**Comment by abhisheks008** [comment]
@yagyesh-bobde Some changes need to be done.

File: MNIST Dataset Classification/requirements.txt (hunk 3 above)
Comment (yagyesh-bobde) [reply MNIST Dataset Classification/requirements.txt]: kk .... i did all the changes can u check it

**Review by abhisheks008 — APPROVED** [review]

**Comment by abhisheks008** [comment]
@yagyesh-bobde Congratulations on your first contribution to this project 🎉

**Pull request merged**